cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js

```diff
@@ -0,0 +1,260 @@
+import uuid from 'uuid';
+import TestTypes from '../../support/dictionary/testTypes';
+import devTeams from '../../support/dictionary/devTeams';
+import permissions from '../../support/dictionary/permissions';
+import UserEdit from '../../support/fragments/users/userEdit';
+import TopMenu from '../../support/fragments/topMenu';
+import SettingsMenu from '../../support/fragments/settingsMenu';
+import generateItemBarcode from '../../support/utils/generateItemBarcode';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import PatronGroups from '../../support/fragments/settings/users/patronGroups';
+import Location from '../../support/fragments/settings/tenant/locations/newLocation';
+import Users from '../../support/fragments/users/users';
+import SearchPane from '../../support/fragments/circulation-log/searchPane';
+import CirculationRules from '../../support/fragments/circulation/circulation-rules';
+import NoticePolicyApi from '../../support/fragments/circulation/notice-policy';
+import NoticePolicyTemplateApi from '../../support/fragments/circulation/notice-policy-template';
+import NewNoticePolicy from '../../support/fragments/circulation/newNoticePolicy';
+import NewNoticePolicyTemplate from '../../support/fragments/circulation/newNoticePolicyTemplate';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import getRandomPostfix from '../../support/utils/stringTools';
+import RequestPolicy from '../../support/fragments/circulation/request-policy';
+import NewRequest from '../../support/fragments/requests/newRequest';
+import Requests from '../../support/fragments/requests/requests';
+
+describe('Request notice triggers', () => {
+  let addedCirculationRule;
+  const patronGroup = {
+    name: 'groupToTestNotices' + getRandomPostfix(),
+  };
+  let userData;
+  const itemData = {
+    barcode: generateItemBarcode(),
+    title: `Instance ${getRandomPostfix()}`,
+  };
+  const testData = {
+    userServicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation('autotestReceiveNotice', uuid()),
+    ruleProps: {},
+  };
+  const createNoticeTemplate = (noticeName) => {
+    return {
+      name: `Autotest_${getRandomPostfix()}_${noticeName}`,
```

Comment:
let's use the method getTestEntityValue(Value) from utils/stringTools.js

Reply:
done